meta.json

```diff
@@ -30,6 +30,12 @@
         "short_desc": "來源：Trainers' Legend G (tinyurl.com/TLGCredits), 官方. 貢獻者：yotv2000tw (+維護者), seanpai96, asps76482, _akito@ytb, project90418512@twt, …"
     },
     {
+        "name": "UmaChimiID ( Indonesian + Local )",
```

Comment:
The name should be the language, followed by the project and other specifics in brackets. Refer to the UmaTL examples.